recipe/meta.yaml

```
@@ -54,10 +53,12 @@ outputs:
       build:
         - gnuconfig  # [unix]
         - {{ compiler('c') }}                # [not win]
+        - {{ stdlib("c") }}                  # [not win]
         - {{ compiler('cxx') }}              # [not win]
         - {{ compiler('fortran') }}          # [not win]
         - gfortran_impl_{{ build_platform }} {{ fortran_compiler_version }}  # [(build_platform != target_platform) and linux]
         - {{ compiler('m2w64_c') }}          # [win]
+        - {{ stdlib("c") }}                  # [win]
```

We should either do
```suggestion
```
or
```suggestion
        - {{ stdlib("m2w64_c") }}                  # [win]
```
and define `m2w64_c_stdlib`.

cc @h-vetinari

---

What package/version should `{{ stdlib("m2w64_c") }}` point to? Also, do we want to (resp. have to) match the `m2w64_` pattern here? Why could feedstock using the m2w64-stack not do a local override in CBC à la:
```
c_stdlib:   # [win]
  - ...     # [win]
```
?

---

We can point do
```yaml
m2w64_c_stdlib:
- m2w64-toolchain
```

> Why could feedstock using the m2w64-stack not do a local override in CBC à la:

Because we want to be able to control it via conda-forge-pinning-feedstock and migrations.

---

Opened a PR: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5832

---

Done